examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/unsafe/DemoUtils.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.example.unsafe;
+
+import io.opentelemetry.sdk.trace.MultiSpanProcessor;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.TracerSdk;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.List;
+
+/**
+ * These methods use reflection to access private fields of the tracer. This is meant as
+ * demonstration only. It is SEVERELY discouraged to access private fields since they are shared
+ * among different threads.
+ */
+@SuppressWarnings("unchecked")
+public class DemoUtils {
```

Comment:
Can we just remove this for the moment? Is it a big downgrade in experience?

Reply:
I know this is a _hacky_ way of doing things that will hardly scale with further changes. Unfortunately, I don't have another solution for showing to the end user of the `ConfigureSpanProcessorExample` example which SpanProcessor is initialized. 
Would you suggest to remove the console output for that example along with the `DemoUtils` class?

Reply:
Let's add a DebugString() or ToString() method on the provider SDK that prints this information for debugging purposes :). What do you think?

Reply:
It sounds a very good solution!

Reply:
After a tiny bit of investigation, this requires to add such a method in all SpanProcessor implementation. I will simply add a comment that tells the reader which SpanProcessor is registered.

Reply:
So personally I don't mind having this util class here - I suggest following this up in a PR/issue if you really want it out @bogdandrutu ;)

Reply:
@thisthat you mentioned that you removed this?

Reply:
I removed any reference to it, but not the file itself :sweat_smile: 
Gone for good in fc8d7491